src/lib/Burrito.hs

```diff
@@ -28,7 +28,6 @@
 -- In short, use @parse@ to parse templates and @expand@ to render them.
 module Burrito
   ( Parse.parse
-  , Render.render
```

Comment:
Hmm, I don't think I got that. Could you explain that a little? What is the advantage of having `render` in addition of `show`?

Apart from that, I would just have to add `render` to the exports of `Burrito.Internal.Type.Template` to get what we want, right?

Reply:
One advantage is API stability. Since `Burrito.render` currently exists, I would like to keep it there unless there's a compelling reason to remove it. 

In general I like to have monomorphic versions of polymorphic functions since they can sometimes help with type inference. In this particular case it doesn't matter much since Burrito doesn't expose any polymorphic ways to create templates. But imagine if it implemented a `Read` instance; then you could use `render . read` to round-trip a template. The alternative would be something like `show @Template . read` or `show . (\ t -> t :: Template) . read`.

Reply:
That makes sense, thank you for the explanation! Should be done in the latest commit